src/ui/screens/messaging/ViewMessage.qml

```diff
@@ -139,6 +141,11 @@ MZViewBase {
                             'badgeTextColor': MZTheme.colors.normalLevelMain,
                             'badgeBackgroundColor': MZTheme.colors.normalLevelBackground
                         };
+                        property var extensionBadge: {
+                            'badgeText': MZI18n.InAppMessagingSubscriptionBadge,
```

Comment:
Shouldn't this be something like `MZI18n.InAppMessagingExtensionBadge` ?

Reply:
🤦🏻 I'm not sure what happened here; I fixed it in both the spots that needed it.